packages/types/src/documents/global/config.ts

```diff
@@ -111,18 +111,26 @@ export interface SCIMInnerConfig {
 
 export interface SCIMConfig extends Config<SCIMInnerConfig> {}
 
-export type AIProvider = "OpenAI" | "Anthropic" | "TogetherAI" | "Custom"
+export type AIProvider =
+  | "OpenAI"
+  | "Anthropic"
+  | "AzureOpenAI"
```

Comment:
`"AzureOpenAI"` was missing from this list, it is referenced in several places outside this file.

Reply:
Did the ts not complain about this missing?

Reply:
No, the typing in this area is a bit loose. This PR tightens it up a little, but there's likely still more we could do.